### PR TITLE
Feat/scout 22 seasons module

### PR DESCRIPTION
**Pull request by dejwi**
[SCOUT-22](https://playmakerpro.atlassian.net/browse/SCOUT-22?atlOrigin=eyJpIjoiMzcxZTkwZDI4NjZjNDdhMTk5MTVjNzRhYTQzMDFhMTUiLCJwIjoiaiJ9)

### Task Description
Seasons module
- table
- create/edit form
- translations
- index/create/edit/id pages
- admin role check
- add to admin panel
- set/unset active season in table menu (+ new hooks)

### Additional Notes (optional)
- no filter form as there's literally no filtering for seasons
- depends on order of setting/unsetting active season sometimes a reload is needed to refresh data display

### CODE REVIEW

File: public/locales/en/common.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
   "REGIONS": "Regions",
   "CLUB": "Club",
   "CLUBS": "Clubs",
-  "SEASON": "Seasom",
+  "SEASON": "Season",
```
Comment (jakubfiglak) [inline public/locales/en/common.json:37]: Good catch 🚀

File: public/locales/pl/seasons.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "NO_NAME_ERROR": "Podaj nazwę sezonu",
+  "NO_START_DATE_ERROR": "Podaj datę rozpoczęcia",
+  "NO_END_DATE_ERROR": "Podaj datę zakończenia",
+  "DELETE_CONFIRM_QUESTION": "Czy na pewno chcesz usunąć sezon {{name}}?",
+  "INDEX_PAGE_TITLE": "Baza sezonw",
```
Comment (jakubfiglak) [inline public/locales/pl/seasons.json:6]: Typo (not mine :D)

File: src/components/nav/nav-list.tsx [hunk 3]
```diff
@@ -122,6 +123,11 @@ export const NavList = () => {
             to="/countries"
             text={t('COUNTRIES')}
           />
+          <NavElement
```
Comment (jakubfiglak) [inline src/components/nav/nav-list.tsx:126]: We're gonna be adding more and more of these `NavElements`, I'd start thinking about simplifying the jsx somehow - eg. keeping these items as an array of objects with an interface of something like:

```
label: string
href: string
icon: ReactNode
```

and rendering elements by looping through this array.

It's not necessary to do right now, I'm just saying it's worth thinking about in the future

File: src/services/api/methods/helpers.ts [hunk 4]
```diff
@@ -136,3 +136,27 @@ export async function unlikeDocument<ReturnType>(
   )
   return data
 }
+
+// set active document
```
Comment (jakubfiglak) [inline src/services/api/methods/helpers.ts:140]: Hmm, I'm not sure we need these helpers at all. `toggle-active` is a functionality specific to `seasons` module (doesn't occur in any other module) so in my opinion there's no benefit in writing these extra helper functions, I'd just create those from scratch in seasons service

**Review by jakubfiglak — COMMENTED** [review]
Great job 🚀  Just some minor comments.

Same note as for `comp-age-cat` module - we're gonna need to adjust to some upcoming backend changes (there'll be a separate ticket for this work)

**Comment by dejwi** [comment]
@jakubfiglak please approve

**Review by jakubfiglak — APPROVED** [review]

**Pull request merged**